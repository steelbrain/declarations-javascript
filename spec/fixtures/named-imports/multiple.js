import { x as xx } from 'file-x'
import { y as yy } from 'file-y'
xx()
yy()
