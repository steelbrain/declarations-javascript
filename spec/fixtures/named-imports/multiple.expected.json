[
  {
    "name": null,
    "position": {
      "start": {
        "line": 1,
        "column": 24
      },
      "end": {
        "line": 1,
        "column": 32
      }
    },
    "source": {
      "name": null,
      "filePath": "file-x",
      "position": {
        "start": {
          "line": 1,
          "column": 24
        },
        "end": {
          "line": 1,
          "column": 32
        }
      }
    }
  },
  {
    "name": null,
    "position": {
      "start": {
        "line": 2,
        "column": 24
      },
      "end": {
        "line": 2,
        "column": 32
      }
    },
    "source": {
      "name": null,
      "filePath": "file-y",
      "position": {
        "start": {
          "line": 2,
          "column": 24
        },
        "end": {
          "line": 2,
          "column": 32
        }
      }
    }
  },
  {
    "name": null,
    "position": {
      "start": {
        "line": 3,
        "column": 30
      },
      "end": {
        "line": 3,
        "column": 43
      }
    },
    "source": {
      "name": null,
      "filePath": "some-module",
      "position": {
        "start": {
          "line": 3,
          "column": 30
        },
        "end": {
          "line": 3,
          "column": 43
        }
      }
    }
  },
  {
    "name": "xx",
    "position": {
      "start": {
        "line": 4,
        "column": 0
      },
      "end": {
        "line": 4,
        "column": 2
      },
      "identifierName": "xx"
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 1,
          "column": 14
        },
        "end": {
          "line": 1,
          "column": 16
        },
        "identifierName": "xx"
      }
    }
  },
  {
    "name": "yy",
    "position": {
      "start": {
        "line": 5,
        "column": 0
      },
      "end": {
        "line": 5,
        "column": 2
      },
      "identifierName": "yy"
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 2,
          "column": 14
        },
        "end": {
          "line": 2,
          "column": 16
        },
        "identifierName": "yy"
      }
    }
  }
]